src/app/pages/ArticlePage/experimentTopStories/helpers.tsx

```diff
@@ -0,0 +1,76 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { OptimoBlock } from '#app/models/types/optimo';
+import { TopStoryItem } from '#app/pages/ArticlePage/PagePromoSections/TopStoriesSection/types';
+import TopStoriesSection from '../PagePromoSections/TopStoriesSection';
+import styles from '../ArticlePage.styles';
+
+export const experimentTopStoriesConfig = {
+  topStoriesExperiment: {
+    variants: {
+      control: 50,
+      show_at_halfway: 50,
+    },
+  },
+};
+
+export const enableExperimentTopStories = ({
+  isAmp,
+  service,
+  pathname,
+}: {
+  isAmp: boolean;
+  service: string;
+  pathname: string;
+}) => {
+  if (!isAmp || !service || !pathname) return false;
+  const urn = pathname.split('/').pop()?.slice(0, -4); // .slice() to remove '.amp' at the end of pathname
+  const newsTestAsset = 'c6v11qzyv8po';
+  const newsAsset = 'cz7xywn940ro';
+  const sportAsset = 'cpgw0xjmpd3o';
```

Comment:
**Question** I assume this is for testing only?

Reply:
For this initial PR yes. I thought it would be a good idea to just keep these restricted to one asset from each service we're impacting just to be extra safe. Will remove this in the following PR once we're happy with it all.